const e=JSON.parse('{"key":"v-6b14a366","path":"/essay/how-to-catch-error.html","title":"에러 처리 정리","lang":"ko-KR","frontmatter":{"title":"에러 처리 정리","description":"2019년 5월 23일에 진행 중인 프로젝트에서 에러 처리 방법을 고민하면서 정리한 포스트입니다. 글의 목적 에러 처리는 소프트웨어의 비정상적인 동작에 대한 큰 사고로 이어지지 않도록 예방하는 것이다. 서비스 운영에 있어서는 사용자들이 서비스를 비정상적으로 사용하는 부분이 있는 지 감지를 할 수 있다. 오랜 시간동안...","head":[["meta",{"property":"og:url","content":"https://the-next-web-research-lab.github.io/essay/how-to-catch-error.html"}],["meta",{"property":"og:site_name","content":"더넥스트웹리서치랩"}],["meta",{"property":"og:title","content":"에러 처리 정리"}],["meta",{"property":"og:description","content":"2019년 5월 23일에 진행 중인 프로젝트에서 에러 처리 방법을 고민하면서 정리한 포스트입니다. 글의 목적 에러 처리는 소프트웨어의 비정상적인 동작에 대한 큰 사고로 이어지지 않도록 예방하는 것이다. 서비스 운영에 있어서는 사용자들이 서비스를 비정상적으로 사용하는 부분이 있는 지 감지를 할 수 있다. 오랜 시간동안..."}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:image","content":"https://the-next-web-research-lab.github.io/images/white_logo.png"}],["meta",{"property":"og:locale","content":"ko-KR"}],["meta",{"property":"og:updated_time","content":"2023-06-22T07:50:41.000Z"}],["meta",{"name":"twitter:card","content":"summary_large_image"}],["meta",{"name":"twitter:image:alt","content":"에러 처리 정리"}],["meta",{"property":"article:modified_time","content":"2023-06-22T07:50:41.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"에러 처리 정리\\",\\"image\\":[\\"https://the-next-web-research-lab.github.io/images/white_logo.png\\"],\\"dateModified\\":\\"2023-06-22T07:50:41.000Z\\",\\"author\\":[]}"]]},"headers":[{"level":2,"title":"글의 목적","slug":"글의-목적","link":"#글의-목적","children":[]},{"level":2,"title":"목차","slug":"목차","link":"#목차","children":[]},{"level":2,"title":"실패를 전달하는 방법","slug":"실패를-전달하는-방법","link":"#실패를-전달하는-방법","children":[{"level":3,"title":"반환값으로 실패 전달","slug":"반환값으로-실패-전달","link":"#반환값으로-실패-전달","children":[]},{"level":3,"title":"try-catch 에러 처리","slug":"try-catch-에러-처리","link":"#try-catch-에러-처리","children":[]},{"level":3,"title":"throw 명시적 선언","slug":"throw-명시적-선언","link":"#throw-명시적-선언","children":[]}]},{"level":2,"title":"어떻게 에러 처리를 해야 할까","slug":"어떻게-에러-처리를-해야-할까","link":"#어떻게-에러-처리를-해야-할까","children":[{"level":3,"title":"컨테이너","slug":"컨테이너","link":"#컨테이너","children":[]},{"level":3,"title":"같은 타입 반환","slug":"같은-타입-반환","link":"#같은-타입-반환","children":[]},{"level":3,"title":"async 함수로 에러 처리","slug":"async-함수로-에러-처리","link":"#async-함수로-에러-처리","children":[]}]},{"level":2,"title":"종합적인 결론","slug":"종합적인-결론","link":"#종합적인-결론","children":[]},{"level":2,"title":"끝","slug":"끝","link":"#끝","children":[{"level":3,"title":"참고자료","slug":"참고자료","link":"#참고자료","children":[]}]}],"git":{"updatedTime":1687420241000},"filePathRelative":"essay/how-to-catch-error.md","autoDesc":true}');export{e as data};
